package.json

```diff
@@ -10,6 +10,9 @@
     "Suhas Karanth <sudo.suhas@gmail.com>"
   ],
   "bin": "index.js",
+  "engines": {
+    "node": ">=8.6.0"
```

Comment:
That’s was intentional not to have it here since it would break CIs with lower versions of Node installed without a particular reason. I’m on my mobile so I can’t search for the issue but please look for it.

Reply:
I see, too bad we can't have optional dev deps :disappointed: 

We should at least pass the name to `node-please-upgrade` to avoid broken message.

Reply:
Fixed.